server/controllers/temporary-accommodation/manage/bookingSearchController.ts

```diff
@@ -1,20 +1,33 @@
 import type { Request, RequestHandler, Response } from 'express'
 import { BookingSearchService } from 'server/services'
-import type { BookingSearchStatus } from '@approved-premises/api'
+import type { BookingStatus } from '@approved-premises/api'
 import extractCallConfig from '../../../utils/restUtils'
 import { createSideNavArr, createTableHeadings } from '../../../utils/bookingSearchUtils'
 
 export default class BookingSearchController {
   constructor(private readonly bookingSearchService: BookingSearchService) {}
 
-  index(status: BookingSearchStatus): RequestHandler {
+  index(status: BookingStatus): RequestHandler {
     return async (req: Request, res: Response) => {
       const callConfig = extractCallConfig(req)
 
       const bookingTableRows = await this.bookingSearchService.getTableRowsForFindBooking(callConfig, status)
 
+      let uiStatus: string
+
+      switch (status) {
```

Comment:
I'm probably missing something but it would be a lot easier if we could just use 'arrived' and 'departed' as the status instead of 'active' and 'closed', especially as this is what it will say in the table headings and side nav

Reply:
I thought as much. 
I think it would make most sense if the functions that did the translating were in the same place and the translations happened at the same time so that if we have to change them in future it's easy

Reply:
good shout, done